### DISCUSSION TITLE
SSL setup for Sec Onion and Winlogbeats

### DISCUSSION THREAD

**Discussion by ColeVan** [Security-Onion-Solutions/securityonion#2145] [Unsupported Versions] [answered]
Hello! I'm aware of how to configure the winlogbeat.yml file for SSL. I am having trouble figuring out how to set up Security Onion with the proper cert and which follow on cert to send to my endpoints to establish a secure connection when it comes to sending hosts logs. 

Are there any blogs, forum posts, githubs, or writeups that I could refer to? Any guidance would be much appreciated. Thanks!

**Comment by endpointsec** [comment, marked as answer]
I can dig for the resources but you can generate the certificate on the box using openssl. You then need to put that crt file into '/opt/so/saltstack/local/salt/logstash/etc/certs' (if I recall correctly). Copy the file to your winlogbeat clients and specify it in the YML file. Test winlogbeat connectivity with 'test output' to ensure the TLS handshake has no problems.
